testsuite/run_sets/finishing.yml

```diff
@@ -8,7 +8,7 @@
 ## Post run features BEGIN ##
 
 # IMMUTABLE ORDER
-
+- features/secondary/srv_rename_hostname.feature
 # this feature is destructive for other features, so it is better at the end
```

Comment:
```suggestion
# this feature is destructive for other features, so it is better at the end
- features/secondary/srv_rename_hostname.feature
```

Small nitpick since FMPOV the description should be above the actual line.